### PR TITLE
Update Ansible deployment content

### PR DESCRIPTION
**Pull request by lilin90**
Via https://github.com/pingcap/docs-cn/pull/471 and https://github.com/pingcap/docs-cn/pull/482.
PTAL

### CODE REVIEW

File: op-guide/ansible-deployment.md [hunk 1]
```diff
@@ -87,13 +90,21 @@ For more information, see [Ansible Documentation](http://docs.ansible.com/ansibl
 
 ## Download TiDB-Ansible to the Control Machine
 
-Use the following command to download the TiDB-Ansible `release-1.0` branch from GitHub [TiDB-Ansible project](https://github.com/pingcap/tidb-ansible).
-The default folder name is `tidb-ansible`. The `tidb-ansible` directory contains all files you need to get started with TiDB-Ansible.
+Use the following command to download the corresponding version of TiDB-Ansible from GitHub [TiDB-Ansible project](https://github.com/pingcap/tidb-ansible). The default folder name is `tidb-ansible`. The `tidb-ansible` directory contains all files you need to get started with TiDB-Ansible.
+
+Download the GA version:
```
Comment (queenypingcap) [inline op-guide/ansible-deployment.md]: GA -> 1.0

File: op-guide/ansible-deployment.md [hunk 2]
```diff
@@ -87,13 +90,21 @@ For more information, see [Ansible Documentation](http://docs.ansible.com/ansibl
...
 git clone -b release-1.0 https://github.com/pingcap/tidb-ansible.git
 ```
 
+Download the master version:
+
+```
+git clone https://github.com/pingcap/tidb-ansible.git
+```
+
+> **Note:** For the production environment, download the GA version to deploy TiDB.
```
Comment (queenypingcap) [inline op-guide/ansible-deployment.md]: GA -> 1.0

File: op-guide/ansible-deployment.md [hunk 3]
```diff
@@ -209,6 +220,26 @@ location_labels = ["host"]
 
     - `capacity`: (DISK - log space) / TiKV instance number (the unit is GB)
 
+### Adjustment of inventory.ini variables
+
+| Variable | Description |
+| ---- | ------- |
+| cluster_name | the name of a cluster, adjustable |
+| tidb_version | the version of TiDB, configured by default in TiDB-Ansible branches |
```
Comment (queenypingcap) [inline op-guide/ansible-deployment.md]: Does “branches” here stand for the master version or the 1.0 version?  If it is, please change branches to versions.

File: op-guide/ansible-deployment.md [hunk 4]
```diff
@@ -209,6 +220,26 @@ location_labels = ["host"]
 
     - `capacity`: (DISK - log space) / TiKV instance number (the unit is GB)
 
+### Adjustment of inventory.ini variables
+
+| Variable | Description |
+| ---- | ------- |
+| cluster_name | the name of a cluster, adjustable |
```
Comment (queenypingcap) [inline op-guide/ansible-deployment.md]: adjustable -> configurable?

File: op-guide/ansible-deployment.md [hunk 5]
```diff
@@ -209,6 +220,26 @@ location_labels = ["host"]
...
+| set_timezone | to edit the timezone of the managed node, True by default; False means closing |
+| enable_elk | currently not supported |
+| enable_firewalld | to enable the firewall, closed by default |
+| enable_ntpd | to monitor the NTP service of the managed node, True by default; do not close it |
+| machine_benchmark | to monitor the disk IOPS of the managed node, True by default; do not close it |
+| set_hostname | to edit the hostname of the mananged node based on the IP, False by default |
+| enable_binlog | whether to deploy Pump and enable the binlog, False by default, dependent on the Kafka cluster; see the `zookeeper_addrs` variable |
+| zookeeper_addrs | the zookeeper address of the binlog Kafka cluster |
+| enable_slow_query_log | to record the slow query log of TiDB into a single file: ({{ deploy_dir }}/log/tidb_slow_query.log). False by default, to record it into the TiDB log |
+| deploy_without_tidb | the KV mode, deploy only PD, TiKV and the monitoring service, not TiDB; set the IP of the tidb_servers host group to null in the `inventory.ini` file |
```
Comment (queenypingcap) [inline op-guide/ansible-deployment.md]: KV -> Key-Value

File: op-guide/ansible-deployment.md (hunk 3 above)
Comment (lilin90) [reply op-guide/ansible-deployment.md]: The "branches" here means the actual branches of a GitHub repo, so just keep it.

**Comment by lilin90** [comment]
@queenypingcap PTAL

**Comment by LinuxGit** [comment]
LGTM

**Pull request merged**